implementations/rust/ockam/ockam_api/src/cloud/addon.rs

```diff
@@ -23,15 +23,83 @@ pub struct Addon<'a> {
     pub enabled: bool,
 }
 
+#[derive(Encode, Decode, Serialize, Deserialize, Debug)]
+#[rustfmt::skip]
+#[cbor(map)]
+pub struct ConfluentConfig<'a> {
+    #[cfg(feature = "tag")]
+    #[serde(skip)]
+    #[cbor(n(0))] pub tag: TypeTag<1697996>,
+
+    #[serde(borrow)]
+    #[cbor(b(1))] pub bootstrap_server: CowStr<'a>,
+
+    #[serde(borrow)]
+    #[cbor(b(2))] pub api_key: CowStr<'a>,
+
+    #[serde(borrow)]
+    #[cbor(b(3))] pub api_secret: CowStr<'a>,
+}
+
+impl<'a> ConfluentConfig<'a> {
+    pub fn new<S: Into<CowStr<'a>>>(bootstrap_server: S, api_key: S, api_secret: S) -> Self {
+        Self {
+            #[cfg(feature = "tag")]
+            tag: TypeTag,
+            bootstrap_server: bootstrap_server.into(),
+            api_key: api_key.into(),
+            api_secret: api_secret.into(),
+        }
+    }
+}
+
+#[derive(Encode, Decode, Serialize, Deserialize, Debug)]
+#[rustfmt::skip]
+#[cbor(map)]
+pub struct ConfluentConfigResponse<'a> {
+    #[cfg(feature = "tag")]
+    #[serde(skip)]
+    #[cbor(n(0))] pub tag: TypeTag<6434816>,
+
+    #[serde(borrow)]
+    #[cbor(b(1))] pub bootstrap_server: CowStr<'a>,
+
+    #[serde(borrow)]
+    #[cbor(b(2))] pub api_key: CowStr<'a>,
```

Comment:
After the discussion during the last demo I have removed the `api_key` from the response schema